src/stories/Library/links/arrow-link/arrow-link.scss

```diff
@@ -0,0 +1,7 @@
+.arrow-link {
+  display: block;
+  text-decoration: none;
+  &__text {
+    margin-bottom: -5px;
```

Comment:
Uhh.. Why are you doing this? As far as i know, a minus margin on bottom wont do anything :)

If you're trying to center the text, then i'd prefer you use flexbox on the actual .arrow-link